api/remote_test.go

```diff
@@ -12,6 +14,13 @@ func TestRemoteHandlers(t *testing.T) {
 		{"POST", "/", mustFile(t, "testdata/postRemoteRequest.json")},
 	}
 
+	// Reject all dag.Info's
+	lib.Config.API.RemoteAlwaysAccept = false
```

Comment:
Hrm, I'm wondering if we can adjust `remoteAlwaysAccept` to instead be `remoteAcceptSizeMax`, and only allow either 0 (`remoteAlwaysAccept == false`) or -1 (`remoteAlwaysAccept == true`). We could later use that to replace the `allowedDagSize` const, but let's do that as a follow up.